.vscode/settings.json

```diff
@@ -1,8 +1,6 @@
 {
   "editor.formatOnSave": true,
   "editor.formatOnPaste": true,
-  "cSpell.ignoreWords": [
-    "ecotricity",
-    "sunmills"
-  ]
-}
\ No newline at end of file
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Added this to automatically invoke prettier on save. This streamlines my DX but can definitely pull this out if undesired!